### ISSUE TITLE
Remove extensions.legacy.enabled and some other config from the readme

### ISSUE THREAD

**Issue by fzzzy** [mozilla/testpilot#3599]
extensions.legacy.enabled was required when we were not a webextension.

testpilot.env is no longer required after https://github.com/mozilla/testpilot/pull/3556 lands.

Setting extensions.install.requireBuiltInCerts is no longer required

Setting xpinstall.signatures.required is no longer needed.

The only pref we need now is extensions.webapi.testing!

**Comment by ghost** [comment]
> testpilot.env is no longer required after #3556 lands.

That should be part of that patch then.  /cc @meandavejustice
